styles.css

```diff
@@ -1063,6 +1149,54 @@ textarea:focus {
   .heading {
     margin-top: 120px;
   }
+
+  .modal-container {
+    background: #fff;
+    border: 1px solid #ebecf0;
+    box-shadow: 48px 48px rgba(37, 47, 137, 0.08);
+    height: 794px;
+  }
+
+  /* .popup-content {
+   display: flex;
+   flex-direction: row;
+   margin-top: 12px;
+   gap: 12px;
+  } */
+
+  .description {
+    width: 311px;
+    height: 336px;
+    font-size: 16px;
+    line-height: 30px;
+    color: #344563;
+    gap: 12px;
+  }
```

Comment:
- [ ] [REQUIRED] Good job for implementing changes requested by previous reviewers you are almost there :+1: But as mentioned early by the previous reviewers you need to stick as much as possible to the Figma template. Please use the same `width`,  `height`, and `spacing` as defined on the Figma template because is very important for the user to have a great user experience and for you to peer attention to details: smile

> * [ ]  Like the previous reviewer mentioned please make sure your design looks as close as possible to the Figma design pray
> 
> Figma Design	Your Design
> ![image](https://user-images.githubusercontent.com/74814780/180230869-fcf27ba8-838d-48a9-a880-ad47fc1a4fc0.png)	![image](https://user-images.githubusercontent.com/74814780/180230955-7aa1ac6d-9522-4d9a-95f5-698f41dc9388.png)

### Figma template
![Screenshot from 2022-07-21 18-17-42](https://user-images.githubusercontent.com/6615394/180285446-cd046e27-432f-4d70-9d62-94b8d226ffc9.png)

### Your popup
![Screenshot from 2022-07-21 18-10-11](https://user-images.githubusercontent.com/6615394/180285453-4bc1b58e-bcbe-4446-8f22-78cc811a1d43.png)